src/Stripe.net/Services/Plans/StripePlanCreateOptions.cs

```diff
@@ -16,20 +16,23 @@ public class StripePlanCreateOptions : StripeBaseOptions
 
         [JsonProperty("interval")]
         public string Interval { get; set; }
-        
+
         [JsonProperty("interval_count")]
         public int? IntervalCount { get; set; }
 
-        [JsonProperty("name")]
-        public string Name { get; set; }
+        [JsonProperty("product")]
+        public StripeProductNestedCreateOptions Product { get; set; }
+
+        [JsonProperty("product")]
+        public string ProductId { get; set; }
 
         [JsonProperty("trial_period_days")]
         public int? TrialPeriodDays { get; set; }
 
-        [JsonProperty("statement_descriptor")]
-        public string StatementDescriptor { get; set; }
-
         [JsonProperty("metadata")]
         public Dictionary<string, string> Metadata { get; set; }
+
+        [JsonProperty("nickname")]
+        public string Nickname { get; set; }
```

Comment:
should be in alphabetical order if we can

Reply:
Ah. Right. Alphabetical, but metadata last.